### PR TITLE
Jenkins 43935 fix language tag of head to output returns a well formed ietf bcp 47 language tag representing this locale

### PR DESCRIPTION
**Pull request by scherler**
# Description

See [JENKINS-43935](https://issues.jenkins-ci.org/browse/JENKINS-43935).

Exposes as well the static resource of jenkins `data-resurl` in a new method in core-js

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by michaelneale** [comment]
this has a corejs change but the dependency is not bumped (also ATH didn
t seem totaly happy last I looked) - confused?

**Comment by scherler** [comment]
@michaelneale Yeah correct if you want we can bumb up the version and do the version dance, but in the end I did not need that method published and could wait until the next publish cycle. 

Regarding ATH I will have a look, but if we do not validate the language attribute (what would surprise me) the ATH failure shall not be related.

**Comment by michaelneale** [comment]
might just need updating to master as there was a change due to activity tables that the ATH is likely picking up - that is probably all.

**Comment by scherler** [comment]
@michaelneale will do, thanks for the headsup

**Comment by scherler** [comment]
ATH test: https://ci.blueocean.io/blue/organizations/jenkins/ATH-Jenkinsfile/detail/master/2335/pipeline

**Comment by scherler** [comment]
@michaelneale ATH passes 
buc6whmcaaascgs

File: blueocean-core-js/src/js/config.js [hunk 1]
```diff
@@ -85,6 +87,13 @@ export default {
         return (typeof config.jenkinsRootURL === 'string' ? config.jenkinsRootURL : '/jenkins');
     },
 
+    getResourceURL() {
+        if (!config.isLoaded) {
+            this.loadUrls();
+        }
+        return (typeof config.resourceUrl === 'string' ? config.resourceUrl : '/jenkins/static');
```
Comment (michaelneale) [inline blueocean-core-js/src/js/config.js]: for some reason, typeof + string makes me remember some really exotic bug we had... and undefined... wish I could remember.

**Comment by michaelneale** [comment]
@scherler what is IETF_BCP_47 ? as you are multilingual, I will take your word for it that it is the Right Thing To do, but curious. 

Otherwise. LGTM :bee:

File: blueocean-core-js/src/js/config.js (hunk 1 above)
Comment (scherler) [inline blueocean-core-js/src/js/config.js]: @michaelneale the above is a replicate from e.g. `getJenkinsRootURL` https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-core-js/src/js/config.js#L85 I guess that it was the solution to the weird bug ;)

**Comment by scherler** [comment]
@michaelneale https://tools.ietf.org/html/bcp47 is a working paper where you find all references to the related ISO standard. It is a working paper on "best current practise" BCP

# Abstract

   This document describes the structure, content, construction, and
   semantics of language tags for use in cases where it is desirable to
   indicate the language used in an information object.  It also
   describes how to register values for use in language tags and the
   creation of user-defined extensions for private interchange.

**Pull request merged**